pkg/handler/topology.go

```diff
@@ -135,9 +135,9 @@ func getTopologyFlows(cfg *loki.Config, client httpclient.Caller, params url.Val
 	return qr, http.StatusOK, nil
 }
 
-func shouldMergeReporters(metricType constants.MetricType) bool {
-	return metricType == constants.MetricTypeBytes ||
-		metricType == constants.MetricTypePackets
+func shouldMergeReporters(metricType constants.MetricType, deduper loki.Deduper) bool {
+	return !deduper.Merge && deduper.Mark && (metricType == constants.MetricTypeBytes ||
```

Comment:
I don't think it make any sense to have mark `true` and merge `true` at the sametime
while we don't enforce this yet but this is something we should consider 
IMO valid configs
Mark   |  Merge
---------------
  T    |    F
  F    |    T
 --------------
 since we always need to know about dup so both T or both F is invalid, maybe we can just single config knob at the agent side ?

Reply:
Both `merge` and `mark` could be false :upside_down_face: 
But I agree both `true` doesn't make sense on eBPF side. I just wanted to manage every cases here since we have 2 flags for now.

I suggest to keep as is and cleanup the code for enablement in https://issues.redhat.com/browse/NETOBSERV-1459